src/Integration.UnitTests/Suppression/SonarQubeIssuesProviderTests.cs

```diff
@@ -759,12 +790,17 @@ private void VerifyRanOnBackgroundThread(Times expected)
 
         private void VerifyServiceGetIssues(Times expected)
         {
-            mockSqService.Verify(x => x.GetSuppressedIssuesAsync(It.IsAny<string>(), null, It.IsAny<CancellationToken>()), expected);
+            mockSqService.Verify(x => x.GetSuppressedIssuesAsync(It.IsAny<string>(), It.IsAny<string>(), It.IsAny<CancellationToken>()), expected);
```

Comment:
Miscellaneous: this helper method is only interested in the number of times the service was called, not the arguments passed to it, so it should use `It.IsAny<string>()` rather than `null`.